tests/test_importexport_city_export.json

```diff
@@ -1 +1,2 @@
\ No newline at end of file
+[{"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000001",
+  "http://www.osp-core.com/city#name": [{"@value": "Freiburg"}], "http://www.osp-core.com/city#coordinates": [{"@value": "13YOs0RR9300000000000000000000", "@type": "http://www.osp-core.com/types#Vector"}], "@type": ["http://www.osp-core.com/city#City"], "http://www.osp-core.com/city#hasPart": [{"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000002"}]}, {"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000002", "http://www.osp-core.com/city#hasPart": [{"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000003"}], "@type": ["http://www.osp-core.com/city#Neighborhood"], "http://www.osp-core.com/city#coordinates": [{"@value": "13YOs0RR9300000000000000000000", "@type": "http://www.osp-core.com/types#Vector"}], "http://www.osp-core.com/city#name": [{"@value": "Littenweiler"}], "http://www.osp-core.com/city#isPartOf": [{"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000001"}]}, {"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000003", "http://www.osp-core.com/city#coordinates": [{"@value": "13YOs0RR9300000000000000000000", "@type": "http://www.osp-core.com/types#Vector"}], "http://www.osp-core.com/city#isPartOf": [{"@id": "http://www.osp-core.com/cuds#00000000-0000-0000-0000-000000000002"}], "@type": ["http://www.osp-core.com/city#Street"], "http://www.osp-core.com/city#name": [{"@value": "Schwarzwaldstraße"}]}, {"@id": "http://www.osp-core.com/cuba#_serialization", "http://www.w3.org/1999/02/22-rdf-syntax-ns#first": [{"@value": "00000000-0000-0000-0000-000000000001"}]}]
```

Comment:
It's the binary representation of the internal numpy, isn't it?

Reply:
Yes. Good I just wanted to make sure that this was not overlooked because it is an important design decision (kill human readability on RDF file but make life easier for us and make the feature more powerful).